cmd/di.go

```diff
@@ -493,6 +494,18 @@ func (di *Dependencies) bootstrapNetworkComponents(options node.OptionsNetwork)
 	}
 
 	di.NetworkDefinition = network
+	trustedURLs := []string{
+		network.EtherClientRPC,
+		network.MysteriumAPIAddress,
+		network.QualityOracle,
+	}
+	for _, url := range trustedURLs {
+		_, err = firewall.AllowURLAccess(url)
```

Comment:
Can You implement setter for multiple rules to have thin code here
Or `AllowURLAccess(urls ...string)`

Reply:
It's the single place. Is it worth it?

Reply:
OK, challenge accepted